tests/L0CodeSigningSingleFileWithDebugSwitch.ts

```diff
@@ -0,0 +1,38 @@
+import mockanswer = require("azure-pipelines-task-lib/mock-answer");
+import tmrm = require("azure-pipelines-task-lib/mock-run");
+import path = require("path");
+
+let taskPath: string = path.join(__dirname, "../task", "codesigning.js");
+let tmr: tmrm.TaskMockRunner = new tmrm.TaskMockRunner(taskPath);
+
+const secureFileId: string = "THESECUREFILEID";
+const signCertPassword: string = "PASSWORD";
+const timeServer: string = "http://timestamp.digicert.com";
+const hashingAlgorithm: string = "SHA256";
+const fileToSign: string = "DllToSign.dll";
+
+const signToolPath: string = path.normalize(path.join(__dirname, "../task", "signtool.exe"));
+const certificatePath: string = path.join(__dirname, "./test-files", "TestCertificate.pfx");
+const filePaths: string = "DllToSign.dll";
+tmr.setInput("secureFileId", secureFileId);
+tmr.setInput("signCertPassword", signCertPassword);
+tmr.setInput("timeServer", timeServer);
+tmr.setInput("hashingAlgorithm", hashingAlgorithm);
+tmr.setInput("files", filePaths);
+tmr.setVariableName("SECURE_FILE_PATH", certificatePath);
```

Comment:
Depending on if we keep the ui option or switch to a system.variable, we should set the information in this test and check if the required flags are actually set.